src/Sentry.Unity/Integrations/UnityBeforeSceneLoadIntegration.cs

```diff
@@ -7,8 +7,8 @@ internal sealed class UnityBeforeSceneLoadIntegration : ISdkIntegration
     {
         private readonly IApplication _application;
 
-        public UnityBeforeSceneLoadIntegration(IApplication? appDomain = null)
-            => _application = appDomain ?? ApplicationAdapter.Instance;
+        public UnityBeforeSceneLoadIntegration(IApplication? application = null)
+            => _application = application ?? ApplicationAdapter.Instance;
```

Comment:
[C# 8 can help here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator)

```suggestion
            => _application ??= ApplicationAdapter.Instance;
```

When using Rider, it should show you a hint to refactor that (with CMD+Enter or whatever shortcut you opt-ed for)

Reply:
I think in this particular case, it doesn't work because `application` gets passed as an argument and we want to set `_application`?

Reply:
Sorry, I missed that.